src/basics.md

```diff
@@ -851,39 +917,40 @@ fn run() -> Result<()> {
+[`MutexGuard`]: https://doc.rust-lang.org/std/sync/struct.MutexGuard.html
+[`Mutex`]: https://doc.rust-lang.org/std/sync/struct.Mutex.html
 [`Normal`]: https://doc.rust-lang.org/rand/rand/distributions/normal/struct.Normal.html
-[`IndependentSample::ind_sample`]: https://doc.rust-lang.org/rand/rand/distributions/trait.IndependentSample.html#tymethod.ind_sample
-[`Rng::gen_range`]: https://doc.rust-lang.org/rand/rand/trait.Rng.html#method.gen_range
+[`Output`]: https://doc.rust-lang.org/std/process/struct.Output.html
+[`Range`]: https://doc.rust-lang.org/rand/rand/distributions/range/struct.Range.html
+[`Read`]: https://doc.rust-lang.org/std/io/trait.Read.html
 [`Regex::captures_iter`]: https://doc.rust-lang.org/regex/regex/struct.Regex.html#method.captures_iter
 [`Regex::replace_all`]: https://docs.rs/regex/0.2.2/regex/struct.Regex.html#method.replace_all
-[`rand::Rand`]: https://doc.rust-lang.org/rand/rand/trait.Rand.html
-[`Range`]: https://doc.rust-lang.org/rand/rand/distributions/range/struct.Range.html
 [`Regex`]: https://doc.rust-lang.org/regex/regex/struct.Regex.html
+[`Rng::gen_range`]: https://doc.rust-lang.org/rand/rand/trait.Rng.html#method.gen_range
+[`RwLock`]: https://doc.rust-lang.org/std/sync/struct.RwLock.html
+[`Stdio::piped`]: https://doc.rust-lang.org/std/process/struct.Stdio.html
+[`bitflags!`]: https://docs.rs/bitflags/*/bitflags/macro.bitflags.html
+[`digest::Context`]: https://docs.rs/ring/*/ring/digest/struct.Context.html
+[`digest::Digest`]: https://docs.rs/ring/*/ring/digest/struct.Digest.html
+[`rand::Rand`]: https://doc.rust-lang.org/rand/rand/trait.Rand.html
 [`rand::Rand`]: https://doc.rust-lang.org/rand/rand/trait.Rand.html
 [`rand::Rng`]: https://doc.rust-lang.org/rand/rand/trait.Rng.html
 [`rand::thread_rng`]: https://doc.rust-lang.org/rand/rand/fn.thread_rng.html
 [`regex::RegexSetBuilder`]: https://doc.rust-lang.org/regex/regex/struct.RegexSetBuilder.html
 [`regex::RegexSet`]: https://doc.rust-lang.org/regex/regex/struct.RegexSet.html
-[`Output`]: https://doc.rust-lang.org/std/process/struct.Output.html
-[`Command`]: https://doc.rust-lang.org/std/process/struct.Command.html
-[`HashMap`]: https://doc.rust-lang.org/std/collections/struct.HashMap.html
-[`Mutex`]: https://doc.rust-lang.org/std/sync/struct.Mutex.html
-[`RwLock`]: https://doc.rust-lang.org/std/sync/struct.RwLock.html
-[`MutexGuard`]: https://doc.rust-lang.org/std/sync/struct.MutexGuard.html
-[`Mmap::as_slice`]: https://docs.rs/memmap/*/memmap/struct.Mmap.html#method.as_slice
 [`seek`]: https://doc.rust-lang.org/std/fs/struct.File.html#method.seek
-[`digest::Context`]: https://docs.rs/ring/*/ring/digest/struct.Context.html
-[`digest::Digest`]: https://docs.rs/ring/*/ring/digest/struct.Digest.html
 [rand-distributions]: https://doc.rust-lang.org/rand/rand/distributions/index.html
 [replacement string syntax]: https://docs.rs/regex/0.2.2/regex/struct.Regex.html#replacement-string-syntax
```

Comment:
same here

Reply:
I don't understand this one. Are you referring to this code, from your above comment?

```rust
        .arg("-ah")
        .arg(&directory)
```

Reply:
@ludwigpacifici 
Well that was not really descriptive on my part :disappointed: (I was doing 3x reviews concurrently at that time, teaches me not to multitask sorry!). 

What I mean was:
Please split this commit into one editing the example and second one editing the links 
and the "same here" was about making the regex links use wildcard "*" instead of "0.2.2" which happens in two places

Reply:
No worries @budziq, you do a good job 👍